app/src/main/java/org/oppia/app/topic/train/SkillInterface.kt

```diff
@@ -0,0 +1,9 @@
+package org.oppia.app.topic.train
+
+/** Interface to update the selectedSkillList in [TopicTrainFragmentPresenter]. */
+interface SkillInterface {
```

Comment:
Maybe: SkillSelector would be clearer. Using 'interface' in the name feels a bit odd. It's like calling something 'SkillObject'.

Reply:
Actually your comment makes sense. Updated `SkillInterface` to `SkillObject`.